src/integration-test/integration/MappingsClient.test.ts

```diff
@@ -0,0 +1,485 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as chaiAsPromised from "chai-as-promised"
+import { expect, use } from "chai";
+import { MappingsClient, MappingCreate, GroupCreate, GroupPropertyCreate, CalculatedPropertyCreate, CustomCalculationCreate, MappingUpdate, ECProperty, GroupUpdate, CalculatedPropertyUpdate, CustomCalculationUpdate, GroupPropertyUpdate, MappingCopy, DataType, QuantityType, CalculatedPropertyType } from "../../reporting";
+import "reflect-metadata";
+import { AuthorizationCallback, getTestRunId, TestConstants, getTestDIContainer, TestIModelGroup, TestIModelGroupFactory, IModelMetadata, TestAuthorizationProvider, TestIModelCreator, ReusableTestIModelProvider } from "../utils/";
+use(chaiAsPromised);
+
+describe("Mapping Client", () => {
+  const mappingsClient: MappingsClient = new MappingsClient();
+  let accessToken: string;
+
+  const mappingIds: Array<string> = [];
+  let groupId: string;
+  let groupPropertyId: string;
+  let calculatedPropertyId: string;
+  let customCalculationId: string;
+
+  let authorization: AuthorizationCallback;
+  let testIModelGroup: TestIModelGroup;
+  let testIModel: IModelMetadata;
+
+  before( async function () {
+    this.timeout(0);
+
+    const container = getTestDIContainer();
+    
+    const authorizationProvider = container.get(TestAuthorizationProvider);
```

Comment:
I think the whole Test iModel creation could be moved into some common place for all integration tests, because it repeats at least in Extraction and Mapping integration tests.

Reply:
moved iModel creation to a global setup function